L1Trigger/L1TMuonEndCap/src/PrimitiveSelection.cc

```diff
@@ -623,85 +574,6 @@ int PrimitiveSelection::get_index_csc(int tp_subsector, int tp_station, int tp_c
 
 // _____________________________________________________________________________
 // RPC functions
-void PrimitiveSelection::cluster_rpc(const TriggerPrimitiveCollection& muon_primitives, TriggerPrimitiveCollection& clus_muon_primitives) const {
```

Comment:
I guess this is done somewhere else?

Reply:
Yes, all moved to EMTFSubsystemCollector.cc now.